recipes/gandlf/meta.yaml

```diff
@@ -47,7 +47,7 @@ test:
 
 about:
   home: "https://github.com/CBICA/GaNDLF"
-  license: BSD 3-Clause Modification
+  license: BSD-3-Clause
```

Comment:
Since it is modified you need to add some sort of extra thing here using AND and a custom license ref.

Reply:
Sorry, I don't follow. Could you please elaborate?

Reply:
Try something like

`BSD-3-Clause AND LicenseRef-Modified`

Reply:
That's misleading. Try

`LicenseRef-Cbica`

Reply:
Sure @isuruf. w/e is fine by me

Reply:
Added the following: `license: BSD-3-Clause AND LicenseRef-CBICA`

Is that okay?

Reply:
Nope. Please use `LicenseRef-CBICA` only

Reply:
Done.